#!/usr/bin/python3
def raise_exception():
    test = "Type"
    print("Raising a {:d} error".format(test + 15))
